--- webpackDevServer.config.js	2021-04-23 15:05:46.461999000 -0600
+++ webpackDevServer.config.new.js	2021-04-30 10:46:50.439804904 -0600
@@ -96,7 +96,7 @@
     // src/node_modules is not ignored to support absolute imports
     // https://github.com/facebook/create-react-app/issues/1065
     watchOptions: {
-      ignored: ignoredFiles(paths.appSrc),
+        ignored: [ignoredFiles(paths.appSrc), '**/.#*', '**/*~', '**/#*#'],
     },
     https: getHttpsConfig(),
     host,
